app/routes/admin/messages.js

```diff
@@ -5,5 +5,147 @@ const { Route } = Ember;
+  model() {
+    return [{
+      recipient: [
+        {
+          name: 'Organizer'
+        },
+        {
+          name: 'Speaker'
+        }
+      ],
+      trigger      : 'Title1',
+      emailMessage : {
+        subject : 'Email subject1',
+        message : 'Hi, the schedule for session1 has been changed'
+      },
+      notificationMessage: {
+        subject : 'Notification subject1',
+        message : 'Hi, the schedule for session1 has been changed'
+      },
+      sentAt: new Date()
+    },
+    {
+      recipient: [
+        {
+          name: 'Organizer'
+        },
+        {
+          name: 'Speaker'
+        }
+      ],
+      trigger      : 'Title2',
+      emailMessage : {
+        subject : 'Email subject2',
+        message : 'Hi, the schedule for session2 has been changed'
+      },
+      notificationMessage: {
+        subject : 'Notification subject2',
+        message : 'Hi, the schedule for session2 has been changed'
+      },
+      sentAt: new Date()
+    },
+    {
+      recipient: [
+        {
+          name: 'Organizer'
+        },
+        {
+          name: 'Speaker'
+        }
+      ],
+      trigger      : 'Title3',
+      emailMessage : {
+        subject : 'Email subject3',
+        message : 'Hi, the schedule for session3 has been changed'
+      },
+      notificationMessage: {
+        subject : 'Notification subject3',
+        message : 'Hi, the schedule for session3 has been changed'
+      },
+      sentAt: new Date()
+    },
+    {
+      recipient: [
+        {
+          name: 'Organizer'
+        },
+        {
+          name: 'Speaker'
+        }
+      ],
+      trigger      : 'Title3',
+      emailMessage : {
+        subject : 'Email subject3',
+        message : 'Hi, the schedule for session3 has been changed'
+      },
+      notificationMessage: {
+        subject : 'Notification subject3',
+        message : 'Hi, the schedule for session3 has been changed'
+      },
+      sentAt: new Date()
+    },
+    {
+      recipient: [
+        {
+          name: 'Organizer'
+        },
+        {
+          name: 'Speaker'
+        }
+      ],
+      trigger      : 'Title3',
+      emailMessage : {
+        subject : 'Email subject3',
+        message : 'Hi, the schedule for session3 has been changed'
+      },
+      notificationMessage: {
+        subject : 'Notification subject3',
+        message : 'Hi, the schedule for session3 has been changed'
+      },
+      sentAt: new Date()
+    },
+    {
+      recipient: [
+        {
+          name: 'Organizer'
+        },
+        {
+          name: 'Speaker'
+        }
+      ],
+      trigger      : 'Title1',
+      emailMessage : {
+        subject : 'Email subject4',
+        message : 'Hi, the schedule for session4 has been changed'
+      },
+      notificationMessage: {
+        subject : 'Notification subject4',
+        message : 'Hi, the schedule for session4 has been changed'
+      },
+      sentAt: new Date()
+    },
+    {
```

Comment:
You can pass the status of checkboxes from in here. Please add the checkbox status here as it will be coming from the API.